code/parachain/node/src/chain_spec/picasso.rs

```diff
@@ -30,11 +30,13 @@ pub fn genesis_config(
 	if let Some(contract) = option_env!("CW_XC_GATEWAY_WASM_PATH") {
 		contracts.push(contract);
 	}
-	if let Some(_contract) = option_env!("CW_XC_INTERPRETER_WASM_PATH") {
-		// not sure what is going on, but it is weird
-		// Thread 'main' panicked at 'contracts in genesis are valid: Module(ModuleError { index:
-		// 180, error: [4, 0, 0, 0], message: Some("CodeValidation") })',
-		// /build/source/parachain/frame/cosmwasm/src/lib.rs:406 contracts.push(contract);
+	// if let Some(contract) = option_env!("CW_XC_INTERPRETER_WASM_PATH") {
+	// 	see https://github.com/ComposableFi/composable/issues/3952
+	// 	contracts.push(contract);
+	// }
+
+	if let Some(contract) = option_env!("CW_20_BASE_WASM_PATH") {
+		contracts.push(contract);
 	}
 
 	let contracts = contracts
```

Comment:
By the way, this could be made even simpler without having to use if
statements and push calls:

```rust
        let contracts = [
            option_env!("CW_XC_GATEWAY_WASM_PATH"),
            option_env!("CW_20_BASE_WASM_PATH"),
        ]
            .into_iter()
            .filter(core::convert::identity)
            ...
```